app/_data/docs_nav_ee_0.33-x.yml

```diff
@@ -130,3 +130,5 @@
       url: /oidc-google
     - text: OpenID Connect with Auth0
       url: /oidc-auth0
+    - text: Upgrades and Migrations
```

Comment:
We need to place `upgrades-migrations.md` within the 0.33 directory as well to link from here.